Makefile

```diff
@@ -137,6 +137,12 @@ reproducible-prestate:   ## Builds reproducible-prestate binary
 	make -C ./op-program reproducible-prestate
 .PHONY: reproducible-prestate
 
+# Include any files required for the devnet to build and run. This appears to be the only one that's actually needed.
+DEVNET_CANNON_PRESTATE_FILES := op-program/bin/prestate-proof.json
+
+$(DEVNET_CANNON_PRESTATE_FILES):
```

Comment:
This runs cannon-prestate if any of the files are *missing*. You can't depend on cannon-prestate as it's a phony.